docker/node.dockerfile

```diff
@@ -1,43 +1,46 @@
-FROM node:22-slim AS builder
+FROM node:21-slim AS builder
```

Comment:
WHy did you reduce this?

Reply:
Is weird but using 22 fail to build on weird issues around vendor packages, and ia more weird because was working before, so I do not know but downgrade to 21 was the fastest step to get everything working.

Btw, this solution was before all this changes even, when you report the first time it does not work for you.

My best here is that we was not pointing specific 22 version and the latest one got some change that break it. 

I can try to try every 22.x.x to aee how far I can get it working or just leave on 21 that is working.

What do you want?

Reply:
Got it working over `node:22.5-slim` and `node:22.5-alpine` so revert to `22` version but use a more specific tag to avoid issues again.